cmd/opampsupervisor/supervisor/config/config.go

```diff
@@ -5,8 +5,22 @@ package config
 
 // Supervisor is the Supervisor config file format.
 type Supervisor struct {
-	Server *OpAMPServer
-	Agent  *Agent
+	Server       *OpAMPServer
+	Agent        *Agent
+	Capabilities *Capabilities `mapstructure:"capabilities"`
+}
+
+// Capabilities is the set of capabilities that the Supervisor supports.
+// TODO: Support more capabilities
```

Comment:
I think this TODO can be removed since we have issues for adding capabilities, and they will be added here as they are implemented.